### DISCUSSION TITLE
Would it be possible to give each graphics card its own instant?

### DISCUSSION THREAD

**Discussion by Robbylikes** [RainbowMiner/RainbowMiner#2667] [General]
Are there any other options besides single mining and combo mining? 

I have 8 cards and would like a third option added we're all gpus function/mine separately from the other gpus. The mining program called Hashauger did this be he died I think. Another thing that was beneficial was that when one of your cards took a shit it didn't take out the entire rig just the card not only that you also could tell which card was giving you grief. I would like it very much if you added a third option.

**Comment by RainbowMiner** [comment]
It is in fact already possible. You can define own GPU groups - down to one GPU equals one separate group.
https://github.com/RainbowMiner/RainbowMiner?tab=readme-ov-file#configgpugroupsconfigtxt

**Reply by Robbylikes** [reply]
I would use the shocked emoji face but since this is GitHub it may come off as childish or unprofessional so thank you.

**Reply by Robbylikes** [reply]
I have had success on the change but I noticed that MSI afterburner did not reflect the settings that I have for ocprofiles.config for dual or the algorithm.config. I even changed the miners.config. it seems to make the changes when I use like minded GPUs but not separate. Am I missing something? I've changed all the ocprofiles and MSI profiles and the settings still do not show in MSI afterburner.
